### PR TITLE
Add Get Power Capabilities to Telemetry agent as static telemetry data

### PR DESCRIPTION
**Pull request by nmgaston**
### PULL DESCRIPTION

- Adds the Power Capabilities to Static Telemetry.  
- Sends Static Telemetry attributes to INBS via INBS Cloud Client in the Cloudadapter.

### Impact Analysis

| Info | Please fill out this column |
| ------ | ----------- |
| Jira ticket | NEXMANAGE-1022 |

### CODE MAINTAINABILITY

- [x] Added required new tests relevant to the changes
- [x] Updated Documentation as relevant to the changes
- [ ] PR change contains code related to security
- [ ] PR introduces changes that break compatibility with other modules/services (If YES, please provide description)
- [x] Run `go fmt` or `format-python.sh` as applicable
- [x] Update Changelog
- [ ] Integration tests are passing
- [ ] If Cloudadapter changes, check Azure connectivity manually

# _Code must act as a teacher for future developers_

### CODE REVIEW

File: inbm/proto/common/v1/common.proto [hunk 1]
```diff
@@ -129,6 +129,19 @@ message PostOperation {
 //  }
 }
 
+message StaticTelemetry {
```
Comment (gblewis1) [inline inbm/proto/common/v1/common.proto:132]: could this be made more generic? maybe a set of key/value pairs to match what we do with other clouds?

Comment (gblewis1) [reply inbm/proto/common/v1/common.proto:132]: (i am not sure which is best though--maybe this is)

Comment (nmgaston) [reply inbm/proto/common/v1/common.proto:132]: Let me see what I can do.

**Review by gblewis1 — APPROVED** [review]

**Comment by nmgaston** [comment]
Verified this is working with Azure cloud.  It's receiving the new telemetry static information of Power Capabilities.  It's just not displaying it which we can decide to enable later if desired.

**Pull request merged**